desc/compute/_qs.py

```diff
@@ -301,6 +422,47 @@ def _B_modes(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="Boozer transform modes norm",
+    label="",
+    units="~",
+    units_long="None",
+    description="Inner product norm for boozer modes basis.",
+    dim=1,
+    params=[],
+    transforms={"B": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+)
+def _boozer_modes_norm(params, transforms, profiles, data, **kwargs):
```

Comment:
it is not specific to boozmn, this is the normalization used when you compute the boozer transform in DESC, I just moved it from the compute functions of `B_mn` to its own thing to avoid recomputation

Reply:
Ah OK I think this is needed because we aren't doing the usual `transform.fit()` and instead is more like an integration for each mode number.